packages/components/src/components/marquee/index.tsx

```diff
@@ -106,11 +106,15 @@ export function MarqueeRows({
   className,
   ...rest
 }: MarqueeRowsProps) {
-  const chunkSize = Math.ceil(children.length / rows);
+  const chunkSize = Math.floor(children.length / rows);
```

Comment:
I ended up with an empty chunk bcs of
```
48 / 9 == 5.3333
ceil(5.333) == 6
``` 

<img width="509" alt="image" src="https://github.com/user-attachments/assets/66c0599c-b0d7-4a4d-8608-3056138657fb" />

now the math is correct